projects/packages/forms/src/blocks/shared/components/jetpack-field.js

```diff
@@ -34,7 +34,7 @@ const JetpackField = props => {
 	const template = useMemo( () => {
 		return [
 			[ 'jetpack/label', { label, required, requiredText } ],
-			[ 'jetpack/input', { type } ],
+			[ type === 'slider' ? 'jetpack/input-range' : 'jetpack/input', { type } ],
```

Comment:
Instead of abstracting at this file, other more custom blocks abstract directly at `edit.js`:

https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/blocks/field-file/edit.js
https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/blocks/field-rating/edit.js

It results a bit more boiletplate (which we can abstract to hooks of course) but otherwise wondering if we should stick to one consistent way?

Reply:
I've refactored this field to avoid JetpackField and use custom edit components instead. It largely follows the patterns of the Rating Field and Input now.